packages/chaire-lib-backend/src/models/db/tokens.db.queries.ts

```diff
@@ -68,7 +69,7 @@ const getOrCreate = async (usernameOrEmail: string): Promise<string> => {
         }
         const apiToken = randomUUID();
         const tokenLifespan = new Date();
-        tokenLifespan.setDate(tokenLifespan.getDate() + tokenLifespanDays);
+        tokenLifespan.setDate(tokenLifespan.getDate() +  config.tokenLifespanDays);
```

Comment:
You'll want to handler a default value if the config does not contains it  :)

Reply:
Agreed. Token hsa been added. "||" syntax allows for "config.tokenLifespanDays" to be selected first and "defaultTokenLifespanDays" to be return in case no tokenLifespanDays key is found in config file.